.github/workflows/release.yaml

```diff
@@ -10,43 +10,111 @@ jobs:
   code-quality:
     uses: ./.github/workflows/code-quality.yaml
 
-  pypi-packaging:
-    name: Build and Publish llm-foundry PyPI Package
+  # rm for testing
+  # pypi-packaging:
+  #   name: Build and Publish llm-foundry PyPI Package
+  #   needs:
+  #   - code-quality
+  #   runs-on: linux-ubuntu-latest
+  #   steps:
+  #   - name: Checkout source
```

Comment:
Is this ready for review?

Reply:
was going to uncomment it back in after review, but can uncomment it before hand